test/e2e/autoscaling/horizontal_pod_autoscaling_behavior.go

```diff
@@ -88,9 +89,9 @@ var _ = SIGDescribe("[Feature:HPA] [Serial] [Slow] Horizontal pod autoscaling (n
 
 			ginkgo.By("verifying time waited for a scale down")
 			framework.Logf("time waited for scale down: %s", timeWaited)
-			framework.ExpectEqual(timeWaited > downScaleStabilization, true, "waited %s, wanted more than %s", timeWaited, downScaleStabilization)
+			gomega.Expect(timeWaited).To(gomega.BeNumerically(">", downScaleStabilization), "waited %s, wanted more than %s", timeWaited, downScaleStabilization)
```

Comment:
Someone already noticed that `framework.ExpectEqual(timeWaited > downScaleStabilization, true)` is useless without printing the values in the explanation.

Now that gomega prints the values, we could stop doing that again in the explanation because it's redundant.

But as we have this message, let's keep using it because as it is right now.

Reply:
So shoudl we (1) revert back to to `framework.ExpectEqual` with the message, the way it was before, (2) use gomega with the message or (3) use gomega without the message?

If I understood you right, you prefer option (2)?

Reply:
`gomega.Expect(timeWaited).To(gomega.BeNumerically(">", downScaleStabilization), "waited %s, wanted more than %s", timeWaited, downScaleStabilization)` is fine - don't change anything.